src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,34 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  error: boolean,
```

Comment:
Use an interface instead of a type alias for defining the Props. This provides better IDE support and is more extensible. 💡